modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/DLAdminManagementToolbarDisplayContext.java

```diff
@@ -900,6 +950,19 @@ private long _getRepositoryId() {
 		return _dlAdminDisplayContext.getRepositoryId();
 	}
 
+	private Set<String> _getSelectedAssetTagIds(
+		HttpServletRequest httpServletRequest) {
+
+		if (_assetTagIds != null) {
+			return _assetTagIds;
+		}
+
+		_assetTagIds = SetUtil.fromArray(
```

Comment:
A very small thing: this field name should be renamed too to be consistent with the getter.

Reply:
Mmmmmm
The problem here is that we doing 
`ParamUtil.getStringValues(httpServletRequest, "assetTagId")`

don't you think that request name has the priority on the naming?

Reply:
Ok, as tests have passed let's forward it and cross fingers.